packages/components/src/validated-form-controls/style.scss

```diff
@@ -61,7 +61,7 @@
 	gap: 4px;
 	margin: 8px 0 0;
 	font-family: $font-family-body;
-	font-size: 0.75rem;
+	font-size: $font-size-small;
```

Comment:
No visual changes (if used in Gutenberg, at least).

<img width="311" height="100" alt="Error message in validated controls" src="https://github.com/user-attachments/assets/17e83632-decb-489e-b1a0-fb9751f90d1e" />